articles/active-directory/develop/msal-net-xamarin-android-considerations.md

```diff
@@ -107,6 +106,102 @@ Here's an example of a class that represents the values of the XML file:
   }
 ```
 
+### System Browser Support in Brokered Authentication
+If you are using the system browser for interactive authentication, it is possible you will have configured your application to use brokered authentication when the device does not have broker installed. In this scenario, MSAL will try to authenticate using the default system browser in the device. This will fail out of the box because the redirect URI is configured for broker and the system browser would know how to use it to navigate back to MSAL. To resolve this, you can configure what is known as an intent filter with the broker redirect URI that you used in step four. You will need to modify your application's manifest to add the intent filter as shown below.
+
+```XML
+<!--Intent filter to capture System Browser or Authenticator calling back to our app after sign-in-->
+<activity
+      android:name="microsoft.identity.client.BrowserTabActivity">
+    <intent-filter>
+          <action android:name="android.intent.action.VIEW" />
+          <category android:name="android.intent.category.DEFAULT" />
+          <category android:name="android.intent.category.BROWSABLE" />
+          <data android:scheme="msauth"
+              android:host="Enter_the_Package_Name"
+              android:path="/Enter_the_Signature_Hash" />
+    </intent-filter>
+</activity>
+```
+
+Substitute the package name that you registered in the Azure portal for the `android:host=` value. Substitute the key hash that you registered in the Azure portal for the `android:path=` value. The signature hash should *not* be URL encoded. Ensure that a leading forward slash (`/`) appears at the beginning of your signature hash.
+
+### Android 11 Support
+
+Apps targeting android 10 (API 29) and below were able to query the OS for a list of which packages were available on the device at any given time. In the interest of privacy and security Android 11 is reducing the package visibility to a default list of OS packages and along with the packages that are specified in the app's AndoridManifest.xml file. In order to utilize the system browser and brokered authentication on Android 11 you must first declare these packages so that they will be visible to the app.
```

Comment:
Again, state the developers goal first, *then* the what/why/how. Currently the reader has to read the entire paragraph only to learn its last sentence what this is really about (emphasis mine):

> **In order to utilize the system browser and brokered authentication on Android 11 you must** first declare these packages so that they will be visible to the app.

Same guidance goes for the H2 for this section, as well - what about Android 11 are we trying to support? If I'm scanning, I'll likely miss that this is about brokered auth with the system browser, specifically (because the H2 this H3 resides under is quite a bit further up the page).

> Apps targeting android 10 (API 29)

"Android" with a big-A.